CMake/ExternalProjectDependency.cmake

```diff
@@ -474,18 +474,13 @@ function(_sb_get_external_project_arguments proj varname)
   list(APPEND _ep_arguments LIST_SEPARATOR ${EP_LIST_SEPARATOR})
 
   list(APPEND _ep_arguments CMAKE_GENERATOR ${_sb_CMAKE_GENERATOR})
```

Comment:
Since this file is copied from https://github.com/commontk/Artichoke let's keep the file unmodified.